apps/server/src/api-data/automation/__tests__/automation.service.test.ts

```
@@ -0,0 +1,264 @@
+    let automations = {
+      enabledAutomations: true,
+      enabledOscIn: true,
+      oscPortIn: 8888,
+      automations: [],
+      blueprints: {},
+    };
+    return {
+      getDataProvider: vi.fn().mockImplementation(() => {
+        return {
+          getAutomation: vi.fn().mockImplementation(() => automations),
+          setAutomation: vi.fn().mockImplementation((newData) => (automations = newData)),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)